bin/trace

```diff
@@ -207,7 +207,7 @@ $Source = "";
 # Maximum length of H5TRACE macro line
 # If the ColumnLimit in .clang-format is changed, this value will need to be updated
 #
-my $max_trace_macro_line_len = 110;
+my $max_trace_macro_line_len = 109;
```

Comment:
This line length keeps bin/trace and bin/format_source from fighting over long TRACE macros

Reply:
I feel like there's a +1/-1 that's not respected somewhere then?

Reply:
Right, but I'm unexcited about digging through a rat's nest of Perl to figure out where it's going wrong. I'm just going to turn the formatter off for that one line.